client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -459,7 +459,7 @@ class PluginMeta extends Component {
 
 				{ path &&
 					<CompactCard
-						href={ `${ path }/${ this.props.slug }` }>
+						href={ trimEnd( `${ path }/${ this.props.slug || '' }`, '/' ) }>
```

Comment:
Can we use [`addSiteFragment`](https://github.com/Automattic/wp-calypso/blob/a70587157b33783ba4d5e0e4a1e136680d459d51/client/lib/route/path.js#L50-L64) instead (and fix it, if needed)?